docs/appendix.rst

```diff
@@ -524,8 +509,145 @@ To fix this, install the Python "wheel" package::
 
     pip install wheel
 
+
+this option should not be the preferred one.
+
+
+.. _`Issues with pywbem verson 1.7+, Urllib3 package version 2, and SSL`:
+
+Issues with pywbem verson 1.7+, Urllib3 package version 2, and SSL
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. index:: pair; troubleshooting: OpenSSL
+.. index:: pair; SSL: OpenSSL
+.. index:: pair; UNSUPPORTED_PROTOCOL: OpenSSL
+
```

Comment:
The text between here and the next heading explains background but is not a trouble shooting item. I realized only after re-reading it that the heading level changes for the next few trouble shooting headings. Should we add a heading here, e.g.:
```
Overview
""""""""
```

Reply:
Agreed and done.  That helped it.